imports/server/assets.ts

```diff
@@ -14,14 +14,6 @@ import UploadTokens from './models/UploadTokens';
 import onExit from './onExit';
 import { BlobType } from './schemas/Blob';
 
-// Meteor has no way to import Assets under ES6 modules; it's just always a
-// global that all server code can access.
-declare global {
-  const Assets: {
-    absoluteFilePath(assetPath: string): string | undefined;
-  };
-}
-
```

Comment:
This is declared upstream: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4f4d4c59bbcf562c961b403ee5868f31ed982704/types/meteor/globals/tools.d.ts#L57

I'm not sure why we ever needed an ambient type declaration for it (it seems like it's been there forever), but TS seems happy to pick it up now.